server.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const {injectDecoratorServerSide} = require("@navikt/nav-dekoratoren-moduler/ssr");
+const path = require("path");
+
+const decoratorParams = {
+    env: process.env.DEKORATOR_MILJO || "prod",
+    simple: true,
+    feedback: false,
+    chatbot: false,
+    shareScreen: false,
```

Comment:
Vi har fått et nytt parameter for å skjule "del-skjerm"-knappen, så legger den inn her og fjerner less-koden.